### PR TITLE
Fix typo in HttpLogging doc

### PR DESCRIPTION
**Pull request by shirhatti**
s/AppHttpLogging/AddHttpLogging

### CODE REVIEW

**Comment by shirhatti** [comment]
I was trying to fix a typo in a 6.0 doc, but it failed since the CI job didn't have the lastest SDK 🤦

File: .github/workflows/build-validation.yml [hunk 1]
```diff
@@ -22,8 +22,8 @@ on:
     branches: [ main ]
 
 env:
-  DOTNET_INSTALLER_CHANNEL: 'release/5.0.1xx'
-  DOTNET_DO_INSTALL: 'false'
+  DOTNET_INSTALLER_CHANNEL: 'release/6.0.1xx-preview4'
```
Comment (Rick-Anderson) [inline .github/workflows/build-validation.yml:25]: @gewarren please review .yml changes

**Review by Rick-Anderson — APPROVED** [review]

File: .github/workflows/build-validation.yml (hunk 1 above)
Comment (gewarren) [reply .github/workflows/build-validation.yml:25]: I'm going to defer to @adegeo on this one.

Comment (Rick-Anderson) [reply .github/workflows/build-validation.yml:25]: @shirhatti  can you explain the YML change added to the logging sample?

**Comment by Rick-Anderson** [comment]
> I was trying to fix a typo in a 6.0 doc, but it failed since the CI job didn't have the lastest SDK 🤦


Sorry, I missed that

File: .github/workflows/build-validation.yml (hunk 1 above)
Comment (Rick-Anderson) [reply .github/workflows/build-validation.yml:25]: @adegeo  cancel, we're good.

**Pull request merged**